src/Controller/ActivityController.php

```diff
@@ -82,7 +83,13 @@ public function indexAction($page, Request $request)
      */
     public function createAction(Request $request)
```

Comment:
Please use autowiring:
```
public function createAction(Request $request, ProjectRepository $repository) {
...
}
```
or even better via an additional route:
```
   /**
     * @Route(path="/create/{id}", name="admin_activity_create_with_project", methods={"GET", "POST"})
     */
    public function createAction(Request $request, ?Project $project = null)
```

Reply:
Thanks for this suggestion! I autowired the repositories, but decided against a different route.